### DISCUSSION TITLE
Track moving objects when using PTZ cameras

### DISCUSSION THREAD

**Discussion by nathanbegin** [blakeblackshear/frigate#9283] [Ideas]
Implement a functionality in Frigate that enables automatic adjustment of the camera angle (rotation and/or zoom) to track the detected object. Home location, max rotation and zoom can be set in the config.

**Comment by NickM-27** [comment]
This feature already exists in frigate 0.13 https://deploy-preview-6262--frigate-docs.netlify.app/configuration/autotracking

**Reply by nathanbegin** [reply]
Oh great ! I will look into it. Thank you ! :)
